libraries/dialogs/README.md

```diff
@@ -78,18 +78,44 @@ agreementDialog {
     }
 }.showDialog(supportFragmentManager)
 ```
-# TODOs
+
+* Selection Dialog:
+
+Dialog with list that user can select. There will be checkBox on the left side of each line and user can change the selection. 
+All **Info Dialog** arguments plus these arguments will be applicable to show selection dialogs.
+
+| Field | Type | Description | Default Value |  
+| ------------- |-------------|-------------| ------------- |  
+| `items` | List<Pair<Boolean, String>> | Item list that will be listed on dialog. | null | 
+| `showItemsAsHtml` | Boolean | Item texts will be parsed as Html if this flag setted as true. | false |
+| `onItemSelectedListener` | (DialogFragment, Int) -> Unit | Listener to notify selected index.  | null |
+
+Sample usage:
+```kotlin
+selectionDialog {
+    title = "Selection Dialog Title"
+    content = getContent()
+    items = getItemsAsHtml()
+    showItemsAsHtml = true
+    onItemSelectedListener = { dialog, index -> 
+        dialog.dismiss()
+        onItemSelected(index)
+    }
+}
+```
+
+## TODOs
 * Implement ListDialog. 
-* Implement SelectionDialog
+* ~~Implement SelectionDialog~~
```

Comment:
I think the tildes are added by mistake :)
~~Implement SelectionDialog~~

Reply:
I've added them especially to show that TODO is done.